src/func/language.js

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (C) 2018 Shivam Tripathi
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+// @flow
+
+import type {
+	FormLanguageT as Language, Transaction
+} from './types';
+import {getAddedItems, getRemovedItems} from './set';
+import _ from 'lodash';
+import {camelToSnake} from '../util';
+
+
+export async function updateLanguageSet(transacting: Transaction,
```

Comment:
Why is this receiving the entity rather than a set, like the other functions?

Reply:
This is more consistent with the implementation of such functions on the bookbrainz-site, where the id is passed. Apart from that, personally, I think we should make it so that the the set is extracted in the function concerned as in all the cases till now we have only either set_id or it's a completely entry. It would make for more tightly bound code to have all things handled in one function, and not having to write separate functions to extract first set and then update them.